odpf/assets/common/v1beta2/event.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package odpf.assets.common.v1beta2;
+
+option java_package = "io.odpf.assets.common";
+option java_outer_classname = "EventProto";
+option go_package = "github.com/odpf/proton/assets/common/v1beta2;commonv1beta2";
+
+import "google/protobuf/timestamp.proto";
+
+// Event represents an event in the system.
+// Event is majorly used to represent the state of the system in the form of events.
+// It can be used in any schema which intend to produce events to message bus.
+message Event {
```

Comment:
@ravisuhag do we still need this? it is not being used in `meteor` atm.

Reply:
What about in kafka sink?

Reply:
We could but that is pretty specific use case (only when using kafka sink) and might not represent a metadata.
But then again, I'm not sure where else to put it 🤔 .

Reply:
We can avoid it from asset but then we need to think about how to handle this in kafka sink.